bindings/jupyroot/README.md

```diff
@@ -45,7 +45,7 @@ c.Draw();
 ```
 
 ## Python ROOTbook
-If you prefer to use Python, you can create a new Python 2 / 3 kernel and
+If you prefer to use Python, you can create a new Python 3 kernel and
```

Comment:
```suggestion
If you prefer to use Python, you can create a new Python kernel and
```
Also, maybe here we don't necessarily need to say "Python 3", but I leave it up to you :)